micrometer-core/src/main/java/io/micrometer/core/instrument/binder/system/ProcessorMetrics.java

```diff
@@ -83,6 +91,14 @@ public ProcessorMetrics(Iterable<Tag> tags) {
         Method getCpuLoad = detectMethod("getCpuLoad");
         this.systemCpuUsage = getCpuLoad != null ? getCpuLoad : detectMethod("getSystemCpuLoad");
         this.processCpuUsage = detectMethod("getProcessCpuLoad");
+        this.processCpuTime = detectMethod("getProcessCpuTime");
+        if (Objects.equals(System.getProperty("com.ibm.lang.management.OperatingSystemMXBean.isCpuTime100ns"),
```

Comment:
I'm not an expert in this, but why are we assuming that this is `true` by default?

Reply:
No, the default unit is 1 ns, but may be changed to 100 ns via this system property. See https://eclipse.dev/openj9/docs/api/jdk11/jdk.management/com/ibm/lang/management/OperatingSystemMXBean.html#getProcessCpuTime()